inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -217,6 +217,3 @@ else()
     #                                          PLUGIN_API)
 endif()
 
-
-# ir serialization functional tests variables
-target_compile_definitions(${TARGET_NAME} PRIVATE IR_SERIALIZATION_MODELS_PATH="${CMAKE_CURRENT_SOURCE_DIR}/ir_serialization/models/")
```

Comment:
How are serializer tests working without this line?
As I understand you have moved only exec_graph tests to plugin specific place. 

And you also moved reference IRs. How are other serializer tests working without these models?  If you removed the definition of `IR_SERIALIZATION_MODELS_PATH`

Reply:
As I understand variable with new model location is exposed via funcTestUtils cmake target (INTERFACE keyword) which is imported by ieFuncTests cmake target. @iefode is my reasoning correct?

`target_compile_definitions(${TARGET_NAME} INTERFACE IR_SERIALIZATION_MODELS_PATH="${CMAKE_CURRENT_SOURCE_DIR}/ir_serialization_models/")`

Reply:
@jdanieck You point is correct.
And one more comment. Gays, when i was refactoring the test, i got a questing. As you know we decided to get rid of using IR-files. Is there another way to validate serialization functionality? (For example, create ngraph function -> serialize -> read serialized IR -> compare with source function) @ilyachur @jdanieck Is it possible? What do think?

Reply:
If we have reference IRs it allows to compare with IR from the MO.

If we use the pipeline "create ngraph function -> serialize -> read serialized IR -> compare with source function" we don't have a check with IR, it means that serialized and MO IR formats can be different but all tests will be passed.

Reply:
Ok, got it. Thanks

Reply:
I decided to revert models moving because, cpu tests execute on different machine than ie build machine (and build, unit and ieFuncTests run on the same). And this makes no sense and test will fail :(